freqtrade/templates/sample_short_strategy.py

```diff
@@ -15,7 +15,7 @@
 import freqtrade.vendor.qtpylib.indicators as qtpylib
 
 
-# TODO-lev: Create a meaningfull short strategy (not just revresed signs).
+# TODO: Create a meaningfull short strategy (not just revresed signs).
```

Comment:
Reversed signs is probably good enough for merge, once it's merged a meaningful strategy will show up soon